queue_job/views/queue_job_views.xml

```diff
@@ -107,12 +109,24 @@
                 <field name="eta" />
                 <field name="date_created" />
                 <field name="date_done" />
+                <field name="exec_time" widget="float_time" />
                 <field name="uuid" />
                 <field name="channel" />
                 <field name="company_id" groups="base.group_multi_company" />
             </tree>
         </field>
     </record>
+    <!-- FIXME: this does nothing really useful -->
+    <record id="view_queue_job_graph" model="ir.ui.view">
+        <field name="name">queue.job.graph</field>
+        <field name="model">queue.job</field>
+        <field name="arch" type="xml">
+            <graph string="Jobs">
+                <field name="job_function_id" type="row" />
+                <field name="exec_time" type="measure" />
+            </graph>
+        </field>
+    </record>
```

Comment:
I think a pivot view gives better information:

```
<pivot string="Queue Job">
    <field name="exec_time" type="measure"/>
    <field name="job_function_id" type="row"/>
    <field name="date_done" type="col" interval="week"/>
</pivot>
```

Example:
![Example of pivot view](https://user-images.githubusercontent.com/417223/107512727-84673980-6ba7-11eb-9eb0-9dc5a197d070.png)

Note: in this example, I used `group_operator="avg"` on the field, which I'm not sure we want.

Reply:
The average is really cool on this view as we can see if it grows over time...

Reply:
looks ok. At least is great start!